src/client.js

```diff
@@ -118,9 +118,10 @@ export default class Client {
    * as a function, it must return `Query`, `Mutation`, or `Document` and recieve the client as the only param.
    * @param {Object} [variableValues] The values for variables in the operation or document.
    * @param {Object} [otherProperties] Other properties to send with the query. For example, a custom operation name.
+   * @param {Object} [headers] Aditional headers to be applied on a request by request basis.
```

Comment:
~Aditional~ Additional

Reply:
I'm not sure why the changes are not reflected in this UX, but the updates have been made.